packages/presentation/lib/screen/movie_details/details_bloc.dart

```diff
@@ -56,16 +61,23 @@ class DetailsBlocImpl extends BlocImpl<DetailsScreenArguments, DetailsData>
       isLoading: true,
     );
     final movieId = arguments.movieInfo.ids?.slug;
+
     if (movieId != null) {
-      final listPerson = await _blocGetCast(movieId);
+      final listPerson = await _getCastUseCase(movieId);
+      final listComments = await _getCommentsUseCase(movieId);
```

Comment:
encapsulate 2 api calls in one usecase. let's move more business logic into domain